api/internal/export.js

```diff
@@ -70,7 +70,17 @@ exports.renderArrangement = function (arrangement) {
 exports.renderSnapshot = function (snapshot, containers, items) {
   let output = [];
   let sider = [snapshot.name, 'car', 'driver', 'passenger'];
+
+  // Builds an index sider depending on the size of the largest container
+  const sizeOfLargestContainer = Math.max.apply(Math, containers.map(function(container) {return container.size;}))
+  let indexSider = ["", ""];
```

Comment:
I'm not following why indexSider is declared with ["", ""]. Does [] not work?

Reply:
["" (first empty quotes is to take the place of the snapshot name), "" (second empty quotes is to take the place of the name of the container)], otherwise the numbers will be at the very top, instead of displaced by 2 rows.